src/Tribe/Amalgamator.php

```diff
@@ -159,7 +202,45 @@ private function amalgamate_venues( $venue_ids ) {
 		}
 		global $wpdb;
 		array_map( 'intval', $venue_ids );
-		$keep    = array_shift( $venue_ids );
+		
+		/**
+		 * Filter the venue IDs that should be kept.
+		 *
+		 * @since TBD
+		 *
+		 * @param bool|array Array of the post IDs to keep or boolean false if not defined.
+		 */
+		$keep = apply_filters( 'tribe_amalgamate_venues_keep_venue', false );
+
+		// If false or not array or empty array, then use the default.
+		if (
+			! $keep
+			|| ! is_array( $keep )
+			|| empty ( $keep )
+		) {
+			$keep = array_shift( $venue_ids );
+		} else {
+			// Check if any of the venue IDs is in the $keep.
+			$intersect = array_intersect( $keep, $venue_ids );
```

Comment:
I think this will work, here is my thinking. (Which might be wrong, of course.

Before going into the `if` $keep can be either false or an integer.

If $keep is false, then we take the existing $venue_ids array, grab the first and then $keep will be the first venue id.
(Yes, type changes. I don't know how bad that is.)

if $keep is not false, then it's an integer, and then we do the intersect.

And of course this probably can be improved. :)